config.json

```diff
@@ -1289,6 +1289,15 @@
         "searching"
       ]
     },
+    {
+      "slug": "rest-api",
+      "uuid": "4c89b6a8-0e89-46c9-5a7e-31bfe6a57007",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
I don't think this should be a core exercise at the moment, which means that it has to be unlocked by a core exercise. Let's first come up with a difficulty and some topics, as they will help in finding a suitable core exercise.

Reply:
Markdown could probably be a good core exercise to unlock this one (because of the 'parsing' topic).